openedx/features/user_leads/helpers.py

```diff
@@ -0,0 +1,29 @@
+from django.db import IntegrityError, transaction
+
+from .constants import UTM_PARAM_NAMES
+from .models import UserLeads
+
+
+def get_utm_params(request):
+    request_get_params = request.GET
+    utm_params = {}
+
+    for key in UTM_PARAM_NAMES.values():
+        utm_param = request_get_params.get(key)
+        if utm_param:
+            utm_params.update({key: utm_param})
```

Comment:
I think it would be better to rename `utm_param` to `value` or `utm_param_value` just like we have `key`. As we have dictionary name `utm_params` and `utm_param` shows the intent that it is complete item/key-value-pair of this dictionary like `key: utm_param_value` is a utm_param not only the value is utm_param.

Reply:
This makes sense, thank you!